myapp/src/components/Toy.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export class Toy extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+    render() {
+        return (
+        <div className="toy-item">
+              <span style={{color: 'blue'}}>{this.props.name}</span><div> - </div>
+              <span>Type: {this.props.type}</span><div> - </div>
```

Comment:
daca folosesti "span" pentru un element Label-Valuare, nu prea are sens sa folosesti "<div>" pentru un simplu "-"
Poti folosi tot "span", iar din css sa pui display: flex; pe ".toy-item" incat sa iti aseze elementele 1 sub 1, sau daca vrei in linie "display: flex; flex-direction: column;"